packages/formatters/README.md

```diff
@@ -0,0 +1,19 @@
+# @stoplight/spectral-formatters
+
+This project exposes the available formatters from the CLI for users that perform custom validation through Javascript.
+
+## Usage
+
+```ts
+import {pretty} from '@stoplight/spectral-formatters';
+
+const spectral = new Spectral();
+...
```

Comment:
```suggestion
// ...
```
maybe let's add a comment here to make sure the code run without any changes

Reply:
Done.